src/connect.js

```diff
@@ -122,15 +147,18 @@ function connect(mapStoreStateToProps, mapDispatchToProps, mergeProps, options =
 			/**
 			 * Returns the props data built from the store state, that should be
 			 * passed to the wrapped component.
-			 * @param {!Object}
+			 * @param {bool} storeState
+			 * @return {!object}
 			 * @protected
 			 */
 			getStoreProps_(storeState) {
-				this.storeProps_ = mapStoreStateToProps(
-					storeState,
-					this.props
-				);
-				return this.storeProps_;
+				let {storeProps_} = this;
+				if (!this.wasRendered ||
```

Comment:
Same here, I think it'd be simpler to read as `!this.storeProps_`, as that would show that it's only on the first time.